### PR TITLE
Docs Updates

### LINKED ISSUE #25: Add information to the setup guide indicating URLs to whitelist for download through secure filters
## Story Summary

A customer saw part of their Enclave setup blocked by a web filtering agent:

enclave install error

We need to document in our install guide somewhere or possibly in our FAQ to whitelist the domain `https://release.enclave.io` in order for everything to download correctly.

### LINKED ISSUE #52: Troubleshooting docs don't tell customers how to collect logs for us
If a customer's problem can't be resolved - it would be useful if we had a section in docs to tell a customer how to elevate the logging level (e.g. `enclave loglevel 5`) - tell the customer how long to run at this log level for, how to collect and send us logs and how to revert the log level afterwards (i.e. is it automatically reverted after process restart or needs to be manually dropped?)

### PR DESCRIPTION
**Pull request by enclave-tom**
closes #25 
closes #52

### CODE REVIEW

File: docs/support/troubleshooting.md [hunk 1]
```diff
@@ -212,4 +216,26 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
 
  If you're still having problems after following this checklist, please contact [support@enclave.io](mailto:support@enclave.io) or join one of our [community support](/community-support/) channels to get help and advice.
 
-<small>Last updated May 5, 2022</small>
+
+## Collecting Logs
+In order to help with troubleshooting support may request logs from you this will often come with a `loglevel`. This increases the verbosity of the logs stored by Enclave.
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:221]: Doesn't feel very clear. Might be worth explaining in more detail what log levels do, and the purpose of the steps below. e.g.

"In order to help with troubleshooting,  our support team may ask you to provide logs. The amount of information recorded in a log is determined the by "Log Level", which we may ask you to increase. The following steps outline how to submit logs at a given log level: "

File: docs/support/troubleshooting.md [hunk 2]
```diff
@@ -212,4 +216,26 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
+
+3. Once you're ready to send the logs you can identify the current log file by running `enclave status`
+    Look for this line in the `Local Identity` section at the top of `enclave status`
+    ```
+    Log file location . : C:\Program Files\Enclave Networks\Enclave\Agent\logs\fabric.universe.20230315.pid.5452.log
+    ```
+
+4. From here you can forward this file over to our support team who will be happy to help. 
+
+    !!! note
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:238]: Can probably simplify the message. How does the user know which files (and how many) they need to send?

**Review by enclave-charlie — CHANGES_REQUESTED** [review]

File: docs/support/troubleshooting.md [hunk 3]
```diff
@@ -238,4 +239,15 @@ In order to help with troubleshooting, our support team may ask you to provide l
     !!! note
             Be aware you may you need to provide additional log files. We recommend sending several files around the time of the issue happening, to ensure that we have as much information as possible to resolve your issue.
 
+### Mobile
+On mobile the process is slightly different. 
+
+1. Open the Enclave app and select the hamburger menu in the top left.
+
+2. Select `settings` and enable debug logging. From here use the app as you would normally to reproduce the bug.
```
Comment (enclave-marc-barry) [inline docs/support/troubleshooting.md:247]: Sounds like Enclave needs to be running / started before step 2 will work?

File: docs/support/troubleshooting.md [hunk 4]
```diff
@@ -218,24 +218,36 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
+### Desktop
+In order to help with troubleshooting, our support team may ask you to provide logs. The amount of information recorded in a log is determined the by `Log Level`, which we may ask you to increase. The following steps outline how to submit logs at a given log level:
 
-1. We will both elevate the current log level of the process `enclave loglevel <desired-log-level>`
+1. We will both elevate the current log level of enclave. To do this run `enclave loglevel <desired-log-level>` in a terminal.
 
-2. Now we will set the profile log level to ensure it is retained after a restart of the process. Perform the following steps;
-    - **Windows**: To do this open a command prompt window as administrator `enclave set-config loglevel Debug`
+2. Now we will set the profile log level to ensure it is retained after a restart of the process. This is done to ensure we continue to get the correct log information between restarts. This is useful if for example the issue only happens when starting Enclave or we need to restart Enclave a few times for the issue to manifest. To do this perform the following steps:
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:226]: I'd add a couple more commas here (around "for example" and after "to do this"):

"This is done to ensure we continue to get the correct log information between restarts. This is useful if, for example, the issue only happens when starting Enclave or we need to restart Enclave a few times for the issue to manifest. To do this, perform the following steps:"

File: docs/support/troubleshooting.md [hunk 5]
```diff
@@ -218,24 +218,36 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
+    - **Windows**: Open a command prompt window as administrator `enclave set-config loglevel <desired-log-level>`
 
-    - **Linux/Mac**: `sudo enclave set-config loglevel Debug`
+    - **Linux/Mac**: Run this command in your terminal `sudo enclave set-config loglevel <desired-log-level>`
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:229]: Feels like there should be a semi-colon introducing the command:

"Run this command in your terminal: `sudo enclave set-config loglevel <desired-log-level>`"

File: docs/support/troubleshooting.md [hunk 6]
```diff
@@ -218,24 +218,36 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
     ```
 
 4. From here you can forward this file over to our support team who will be happy to help. 
 
     !!! note
-            Be aware you may you need to provide additional log files for example if the process has restarted frequently, these will be in the same location as described above. The names will vary as the names are generated when Enclave is started.
+            Be aware you may you need to provide additional log files. We recommend sending several files around the time of the issue happening, to ensure that we have as much information as possible to resolve your issue.
+
+### Mobile
+On mobile the process is slightly different.
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:243]: A colon might be more appropriate than a full-stop, as the sentence introduces text:

"On mobile the process is slightly different: "

File: docs/support/troubleshooting.md [hunk 7]
```diff
@@ -218,24 +218,36 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
 4. From here you can forward this file over to our support team who will be happy to help. 
 
     !!! note
-            Be aware you may you need to provide additional log files for example if the process has restarted frequently, these will be in the same location as described above. The names will vary as the names are generated when Enclave is started.
+            Be aware you may you need to provide additional log files. We recommend sending several files around the time of the issue happening, to ensure that we have as much information as possible to resolve your issue.
+
+### Mobile
+On mobile the process is slightly different. 
+
+1. Open the Enclave app ensuring that enclave is first running then select the hamburger menu in the top left.
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:245]: Reckon a few more commas would help here:

"Open the Enclave app, ensuring that enclave is first running, and then select the hamburger menu in the top left."

File: docs/support/troubleshooting.md [hunk 8]
```diff
@@ -218,24 +218,36 @@ If you find an Enclave Gateway isn't working as expected, please start by follow
...
+
+2. Select `settings` and enable debug logging. From here use the app as you would normally to reproduce the bug.
```
Comment (enclave-charlie) [inline docs/support/troubleshooting.md:247]: I'd add a comma after "From here"

**Review by enclave-charlie — CHANGES_REQUESTED** [review]
Looking good, but I'd add a couple more commas to break up the text (I've highlighted them in the review comments)

**Review by enclave-charlie — APPROVED** [review]
Sweet - looks good 👍

**Pull request merged**